docs/config/events.md

```diff
@@ -62,15 +64,46 @@ events:
     - echo "I JUST YARNED"
 ```
 
+An exception for this is events that are based on [tooling](./../tooling.md) commands which will use the tooling `service` as the default.
+
+```yaml
+events:
+  post-thing:
+    - some-command
+tooling:
+  thing:
+    service: web
+```
+
+In the above scenario `some-command` will run on the `web` service by default instead of the `appserver`. For [dynamic tooling routes](./../tooling.md#dynamic-service-commands) events will use the default of the dynamic route.
```

Comment:
In the above scenario, `some-command`...  and For [dynamic tooling routes], events ...